suite-native/receive/src/screens/ReceiveModalScreen.tsx

```diff
@@ -43,8 +46,15 @@ const ReceiveModalScreenSubHeader = ({ accountKey, tokenContract }: ScreenSubHea
     const iconName = tokenContract ?? networkSymbol;
     const coinSymbol = (ethereumTokenSymbol ?? networkSymbol)?.toUpperCase();
 
+    const handleGoBack = () => {
```

Comment:
Use screen events instead, use `useFocusEvent` https://reactnavigation.org/docs/navigation-lifecycle#react-navigation-lifecycle-events instead

Reply:
The blur didn't work but I attached an event listener for removing the screen from stack https://github.com/trezor/trezor-suite/pull/10250/commits/cd8402c5e0a7497e79fb608b8692c434eb0f2d94